vochain/scrutinizer/process.go

```diff
@@ -333,13 +328,17 @@ func (s *Scrutinizer) updateProcess(pid []byte) error {
 	}
 	// Retry if error is "Transaction Conflict"
 	for {
+		s.processUpdateMutex.Lock()
```

Comment:
Why the Lock is inside the loop and not outside?

Reply:
I proposed this because the for is executed per process

Reply:
Does not look correct to me, there is no need for getting and releasing the lock on each iteration. Anyway this is already outated code :)